onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -225,6 +125,153 @@ def append_to_tableau_colulmn_headers(header, question_type=None):
 
         return tableau_colulmn_headers
 
+    def fetch_form_submissions(self, xform, gt_id=None, count_check=False):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6116910)